src/shared/components/query/QueryStore.spec.ts

```diff
@@ -44,7 +44,7 @@ describe("QueryStore", ()=>{
             );
         });
     });
-    describe('Virtual Studies section on query page', () => {
+    describe.skip('Virtual Studies section on query page', () => {
```

Comment:
we will restore these when we can refactor store to not invoke mobx-promise on instantiation